clio-client/src/main/scala/org/broadinstitute/clio/client/webclient/ClioWebClient.scala

```diff
@@ -99,4 +100,33 @@ class ClioWebClient(clioHost: String, clioPort: Int, useHttps: Boolean)(
   ): Future[A] = {
     Unmarshal(httpResponse).to[A]
   }
+
+  def ensureOkResponse(httpResponse: HttpResponse): HttpResponse = {
+    if (httpResponse.status.isSuccess()) {
+      httpResponse
+    } else {
+      throw new Exception(
+        s"Got an error from the Clio server. Status code: ${httpResponse.status}"
+      )
+    }
+  }
+
+  def verifyCloudPaths(config: BaseArgs): Future[Unit] = {
+    Future {
+      config.location.foreach {
+        case "GCP" => ()
+        case _ =>
+          throw new Exception(
+            "Only GCP unmapped bams are supported at this time."
+          )
+      }
+      config.ubamPath.foreach {
```

Comment:
We we run input validations its nice to collect all errors instead of fast failing.. that way you can fix them all instead of one and trying again.